src/gigs-board/components/posts/ProfileHeader.jsx

```diff
@@ -0,0 +1,99 @@
+/* INCLUDE: "common.jsx" */
```

Comment:
Where is `ProfileHeader` component used?

Reply:
I was having issues getting it to load with the avatar in a circle. and was spending too much time on finding a compatible way to achieve it within my limited previous bos building. So I built a header that pulls in the profile image directly that can be formatted easier. and as a bonus the avatar load times dropped from about 30 seconds+ to about 5 seconds.